source/operations/checklists/security.rst

```diff
@@ -8,4 +8,86 @@ Security Checklist
    :local:
    :depth: 2
 
-Stub - detail Security checklist items
+Use the following checklist when planning the security configuration for a production, distributed MinIO deployment.
+
+Required Steps
+--------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Define group policies either on MinIO or the selected 3rd party Identity Provider (LDAP/Active Directory or OpenID)
+
+   * - :octicon:`circle`
+     - Define individual access policies on MinIO or the selected 3rd party Identity Provider
+
+   * - :octicon:`circle`
```

Comment:
This might be a good Conditional, or just mark as **K8s Only**

Reply:
👍